test/helpers-test.js

```diff
@@ -282,6 +282,33 @@ describe("canjs-devtools-helpers", () => {
             });
         });
 
+        describe("can be created with StacheElement", () => {
```

Comment:
Is this test required anymore? Since you updated all the tests to use a Symbol.

Reply:
ah.. that's a good point.